tests/fastcs/panda/test_hdf_panda.py

```diff
@@ -171,6 +171,7 @@ def assert_resource_document():
                     "dataset": f"/{dataset_name}",
                     "swmr": False,
                     "multiplier": 1,
+                    "chunk_size": None,
```

Comment:
The same as above; does `chunk_size is None` means that the leftmost dimension is not chunked (i.e. single chunk)?

Reply:
In this case, yes `None` is intended to mean that the dataset is all in a single chunk

Reply:
Should we make it `0` rather than None?